ai_ref_kits/conversational_ai_chatbot/README.md

```diff
@@ -2,143 +2,195 @@
 
 ```shell
 cd openvino_build_deploy/ai_ref_kits/conversational_ai_chatbot
 ```
 
-### Creating a Virtual Environment
+### Create a Virtual Environment
 
-To create a virtual environment, open your terminal or command prompt and navigate to the directory where you want to create the environment. Then, run the following command:
+To create a virtual environment, open your terminal or command prompt, and go to the directory where you want to create the environment. 
+
+Run the following command:
 
 ```shell
 python3 -m venv venv
 ```
-This will create a new virtual environment named "venv" in the current directory.
+This creates a new virtual environment named "venv" in the current directory.
+
+### Activate the Environment
 
-### Activating the Environment
+The command you run to activate the virtual environment you created depends on whether you have a Unix-based operating system (Linux or macOS) or a Windows operating system.
 
-Activate the virtual environment using the following command:
+To activate  the virtual environment for a **Unix-based** operating system, run:
 
 ```shell
-source venv/bin/activate   # For Unix-based operating systems such as Linux or macOS
+source venv/bin/activate   # This command is for Unix-based operating systems such as Linux or macOS
 ```
 
-_NOTE: If you are using Windows, use `venv\Scripts\activate` command instead._
+To activate the virtual environment for a **Windows** operating system, run:
 
-This will activate the virtual environment and change your shell's prompt to indicate that you are now working within that environment.
+```shell
+source venv\Scripts\activate   # This command is for Windows operating systems
+```
+This activates the virtual environment and changes your shell's prompt to indicate that you are now working in that environment.
 
-### Installing the Packages
+### Install the Packages
 
 To install the required packages, run the following commands:
 
 ```shell
 python -m pip install --upgrade pip 
 pip install -r requirements.txt
 ```
-## How to Access LlaMA
+## Get Access to Llama
+
+_NOTE: If you already have access to the Llama model weights, you can proceed to the authentication step, which is mandatory to convert the Llama model._
+
+## Convert and Optimize the Model
+
+The application uses three separate models. Each model requires conversion and optimization for use with OpenVINO™. The following process includes a step to convert and optimize each model.
 
-_NOTE: If you already have access to the LlaMA model weights, skip to the authentication step, which is mandatory for converting the LlaMA model._
+_NOTE: This reference kit requires more than 8GB of bandwidth and disk space for downloading models. Because of the large model size, when you run the kit for the first time, the conversion can take more than two hours and require more than  32GB of memory. After the first run, the subsequent runs should finish much faster._
```

Comment:
```suggestion
_NOTE: This reference kit requires more than 8GB of bandwidth and disk space for downloading models. Because of the large model size, when you run the kit for the first time, the conversion can take more than two hours and require more than 32GB of memory. After the first run, the subsequent runs should finish much faster._
```